Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/InteractableOnPressReceiver.cs

```diff
@@ -11,33 +12,71 @@ namespace Microsoft.MixedReality.Toolkit.UI
     /// </summary>
     public class InteractableOnPressReceiver : ReceiverBase
     {
+
         [InspectorField(Type = InspectorField.FieldTypes.Event, Label = "On Release", Tooltip = "The button is released")]
         public UnityEvent OnRelease = new UnityEvent();
 
+        public enum PressType
+        {
+            NearAndFar = 0,
+            NearOnly = 1,
+            FarOnly = 2
+        }
+
+        [InspectorField(Label = "Press Interaction Type", Tooltip = "Specify whether press event is for near or far interaction", Type = InspectorField.FieldTypes.DropdownInt, Options = new string[] { "Near and Far", "Near Only", "Far Only" })]
```

Comment:
I am not a fan of the name "Press Interaction Type" Suggestions:

* Near / Far Interaction Filter
* Near or Far Interaction?
* Interaction Filter
* Interaction Distance Filter
* Event Filter Type
* Event Filter

Reply:
I vote for "Interaction Filter"